motion_predict/test/test_predict_ctrv.cpp

```diff
@@ -206,10 +206,10 @@ TEST(predict_ctrv, predictPeriod)
   ASSERT_NEAR(5.1, results[0].predicted_position.position.x, 0.00001);  // Verify update functions were called
   ASSERT_NEAR(0.99, results[0].predicted_position_confidence, 0.01);
   ASSERT_NEAR(0.001, results[0].predicted_velocity_confidence, 0.001);
-  
-  rclcpp::Time new_time = rclcpp::Time(obj.header.stamp) + rclcpp::Duration(int32_t(0.1 * 1e9), 0); // Increase by 0.1 sec
```

Comment:
I am not familiar with 1e9 in this context. Why do we multiply this value? Can we use a constant variable for this if used multiple times in the cpp file?